src/screens/redeem/children/rcTopUp.tsx

```
@@ -1,11 +1,14 @@
+import React, { Fragment, useState, useEffect, useRef, useMemo, useCallback } from 'react';
 import { injectIntl } from 'react-intl';
-import { Text, View, ScrollView } from 'react-native';
+import { Text, View, ScrollView, TouchableOpacity } from 'react-native';
 import { WebView } from 'react-native-webview';
 import { useQuery } from '@tanstack/react-query';
 import { getRcDelegationPricesQueryOptions, getPointsQueryOptions } from '@ecency/sdk';
 import { ScaleSlider } from '../../../components';
 import { hsOptions } from '../../../constants/hsOptions';
+import RootNavigation from '../../../navigation/rootNavigation';
+import ROUTES from '../../../constants/routeNames';
+import { useFocusEffect } from '@react-navigation/native';
```

ESLint

[error] `@react-navigation/native` import should occur before import of `../../../components`

(import/order)